packages/transformers/jsonld/src/JSONLDTransformer.js

```diff
@@ -0,0 +1,110 @@
+// @flow
+
+import {Transformer} from '@parcel/plugin';
+import json5 from 'json5';
+import { isURL, urlJoin } from '@parcel/utils';
+
+// A list of all attributes in a schema that may produce a dependency
+// Based on https://schema.org/ImageObject
+// Section "Instances of ImageObject may appear as values for the following properties"
+const SCHEMA_ATTRS = [
+  'logo',
+  'photo',
+  'image',
+  'thumbnail',
+  'screenshot',
+  'primaryImageOfPage',
+  'embedUrl',
+  'thumbnailUrl',
+  'video',
+  'contentUrl',
+  'url'
+];
+
+export default new Transformer({
+  async transform({ asset, config, logger, options, resolve }) {
+    // allowing any recieved jsonld to be in json5 format
+    let rawCode = await asset.getCode();
+    let jsonCode = json5.parse(rawCode);
+
+    /* console.log(jsonCode);
+    console.log(config);
+    console.log(options);
+    console.log(JSON.stringify({ asset, config, logger, options, resolve })); */
+    
+    console.log(jsonCode);
+    let parser = new JSONLDParser(asset);
+    console.log("parsing....");
+    parser.parse(jsonCode);
+    console.log("done");
+    console.log(jsonCode);
+    
+    // this will send the jsonld to the JSONTransformer
+    asset.type = 'json';
+    // setting it to jsonCode since the parser updates asset paths
+    asset.setCode(JSON.stringify(jsonCode));
+    return [asset];
+  },
```

Comment:
- parcel v1 seems to inject the json object write back into the script tag on the html page
- The only difference is that any asset paths that was in the jsonld are now updated with the new asset paths
- However, my local parcel injects the json object wrapped in javascript code with `module.exports=` and syntax like that

Reply:
Yeah that makes sense... the json transformer will convert it into js.

We have to figure out a way to ignore this in some way, I think we were gonna query parameters for this like in https://github.com/parcel-bundler/parcel/pull/4209

But that PR also implements the query param support so maybe it should get merged asap so this can output json?

Reply:
We could also put the JSON transformer on a named pipeline instead and insert that pipeline to JSON imports in JS (just as we do with url dependencies and `url:`)

Reply:
That's also a possibility and probably a better option in this case (as it will unblock this PR)

Reply:
@DeMoorJasper & @mischnic let me know if there is anything I can do to help get this PR unblocked or if there is another PR I should be keeping my eyes on.

Reply:
You can use named pipelines for now. So nothing should be blocking this PR :)

Reply:
Is there something I need to add to the code here in order to use the named pipelines? Or are you saying the code is fine how it is because the named pipelines will pick it up.

Reply:
change the `"*.{json,json5}": ["@parcel/transformer-json"],` into something like `"js:*.{json,json5}": ["@parcel/transformer-json"],`

Than `js:./something.json` will parse json into js and `url:./something.json` will just copy the json to the output directory and in your case not transform jsonld beyond your transformer